tensorflow/lite/arena_planner.cc

```diff
@@ -284,7 +284,7 @@ TfLiteStatus ArenaPlanner::CalculateTensorDeallocation(int tensor_index) {
 
 TfLiteStatus ArenaPlanner::CalculateAllocationOfInternalTensors(
     int node_index) {
-  if (node_index < graph_info_->num_nodes()) {
+  if (static_cast<size_t>(node_index) < graph_info_->num_nodes()) {
```

Comment:
Let's use `static_cast<int>(graph_info_->num_nodes())` for consistency with casts. See also note above.

Reply:
@jdduke , i have updated the code as per comments, kindly check and update.

Regards
Amit